hack/util.sh

```diff
@@ -301,15 +306,15 @@ function wait_for_url_timed {
 	prefix=${2:-}
 	max_wait=${3:-10*TIME_SEC}
 	wait=0.2
-	expire=$(($(time_now) + $max_wait))
+	expire=$(($(time_now) + max_wait))
 	set +e
 	while [[ $(time_now) -lt $expire ]]; do
-	out=$(curl --max-time 2 -fs $url 2>/dev/null)
+	out=$(curl --max-time 2 -fs "$url" 2>/dev/null)
 	if [ $? -eq 0 ]; then
 		set -e
-		echo ${prefix}${out}
+		echo "${prefix}${out}"
```

Comment:
This change means we won't clean up whitespace (i.e., any newlines in `$out` will be retained, and consecutive blanks will no longer be squeezed), but AFAICS the output of `wait_for_url_timed` is intended only for humans to read anyway, so it shouldn't break anything.